src/eleveldb.erl

```diff
@@ -132,14 +132,14 @@ init() ->
                          {whole_file_expiry, boolean()}
                         ].
 
--type read_option() :: {verify_checksums, boolean()} |
-                       {fill_cache, boolean()} |
-                       {iterator_refresh, boolean()}.
+-type read_options() :: [{verify_checksums, boolean()} |
```

Comment:
This is equivalent... but I preferred the original notation.
But don't worry about it.

[ a | b ] can easily be read as a list with either an a or a b.
whereas list [ a_or_b() ] is a list of a's and b's (plural) and even the empty list is a list that fits.
But, semantically this is equivalent.